src/clj/game/core/installing.clj

```diff
@@ -200,10 +201,85 @@
                       (all-installed state :corp))]
     (concat hosts (server-list state card))))
 
+(defn- corp-install-continue
+  [state side eid card server {:keys [install-state host-card] :as args} slot cost-str]
+  (let [cdef (card-def card)
+        dest-zone (get-in @state (cons :corp slot))
+        install-state (or install-state (:install-state cdef)) 
+        c (-> card
+              (assoc :advanceable (:advanceable cdef) :new true)
+              (dissoc :seen :disabled))]
+    (clear-install-cost-bonus state side)
+    (when-not host-card
+      (corp-install-message state side c server install-state cost-str))
+    (play-sfx state side "install-corp")
+
+    (let [moved-card (if host-card
+                       (host state side host-card (assoc c :installed true))
+                       (move state side c slot))]
+      (when (is-type? c "Agenda")
+        (update-advancement-cost state side moved-card))
+
+      ;; Check to see if a second agenda/asset was installed.
+      (when-completed (corp-install-asset-agenda state side moved-card dest-zone server)
+                      (letfn [(event [state side eid _] (trigger-event-sync state side eid :corp-install (get-card state moved-card)))]
+                        (cond
```

Comment:
As these are all `install-state` comparisons, it might be smart to use a [case function](https://clojuredocs.org/clojure.core/case)?